kratos/tests/sources/test_nodal_data.cpp

```diff
@@ -80,12 +80,10 @@ namespace Kratos {
 					velocity_y_sum += i_node->FastGetSolutionStepValue(VELOCITY_Y);
 				}
 				KRATOS_CHECK_DOUBLE_EQUAL(pressure_sum, i * size);
-				KRATOS_CHECK_DOUBLE_EQUAL(velocity_x_sum, i * size);
+				KRATOS_CHECK_DOUBLE_EQUAL(velocity_x_sum, 2*i * size);
 				KRATOS_CHECK_DOUBLE_EQUAL(velocity_y_sum, 0.00);
 			}
 		}
```

Comment:
The last comment is true. If you have a variable with lexical order between two components name they won't be consecutive. But in the new hash base key (which is not yet in this branch) the components would be forced:
```
		64           size         32-bit hash                comp. index 2-bits
		 |-----------|----|---------------------------------------------|-|
```